addons/note/tests/test_note.py

```diff
@@ -1,25 +1,21 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.tests import common
+from openerp.tests.common import TransactionCase
 
-class TestNote(common.TransactionCase):
+
+class TestNote(TransactionCase):
 
     def test_bug_lp_1156215(self):
         """ensure any users can create new users"""
-        cr, uid = self.cr, self.uid
-        IMD = self.registry('ir.model.data')
-        Users = self.registry('res.users')
 
-        _, demo_user = IMD.get_object_reference(cr, uid, 'base', 'user_demo')
-        _, group_id = IMD.get_object_reference(cr, uid, 'base', 'group_erp_manager')
+        demo_user = self.env.ref('base.user_demo')
+        group = self.env.ref('base.group_erp_manager')
 
-        Users.write(cr, uid, [demo_user], {
-            'groups_id': [(4, group_id)],
-        })
+        demo_user.write({'groups_id': [(4, group.id)]})
 
         # must not fail
-        Users.create(cr, demo_user, {
+        self.env(user=demo_user.id)['res.users'].create({
             'name': 'test bug lp:1156215',
             'login': 'lp_1156215',
         })
```

Comment:
After this creation it should be check that its created or not, Its just suggetion. :sake: